pkg/cli/logfile/storage_entry_otel_test.go

```diff
@@ -22,17 +21,13 @@ import (
 
 func TestTraceRoundtripJSON(t *testing.T) {
 	testFilePath := "testdata/trace.json"
-	originalJSONStr, err := os.ReadFile(testFilePath)
+	originalJSON, err := os.ReadFile(testFilePath)
 	if err != nil {
 		t.Fatalf("unable to read %s: %v", testFilePath, err)
 	}
 
-	var originalJSON bytes.Buffer
-	err = minifyjson.Minify(minify.New(), &originalJSON, bytes.NewReader(originalJSONStr), nil)
-	assert.NilError(t, err, "failed to minify original json")
```

Comment:
The minifying code made it way too difficult to determine what changes were made, so I got rid of it and reformatted the test data via `python3 -m json.tool --indent 2`.